nerf/colmap_provider.py

```diff
@@ -270,13 +270,17 @@ def __init__(self, opt, device, type='train', n_test=24):
                 self.cam_near_far.append([np.min(depth), np.max(depth)])
 
                 # sparse depth info
+                # drop out unconfident and remote points
+                pts_ids = depth < np.min(depth) + (np.max(depth) - np.min(depth))*0.2
+                pts_ids = np.logical_and(weight.astype(np.float32) > 2 - 2*self.opt.sparse_depth_ratio, pts_ids)
```

Comment:
Is there any insight behind this ratio? I think weight already describes the possible estimation error in supervision.

Reply:
found that training with some remote points will harm the convergence of the model when their error is large. The mentioned ratio means the percentage of the depth supervision we shall use as supervision, assuming that the weights are presented in a uniform distribution, although it is a naive assumption. Please inform me in case you have better ideas